src/benchmarks/micro/Program.cs

```diff
@@ -29,8 +29,7 @@ private static IConfig GetConfig()
                     .WithMinIterationCount(15)
                     .WithMaxIterationCount(20) // we don't want to run more that 20 iterations
                     .AsDefault()) // tell BDN that this are our default settings
-                .WithArtifactsPath(Path.Combine(
-                    Path.GetDirectoryName(typeof(Program).Assembly.Location), "BenchmarkDotNet.Artifacts"))
+                .With(ConfigOptions.DontOverwriteResults) // tell BDN to not overwrite the results
```

Comment:
Data generated by the harness should be with the binaries, not drop side-by-side with the source code. I do not agree this should be changed.

Reply:
I believe the opposite. With more than 1.2 million downloads of BenchmarkDotNet where this was always the default setting nobody ever complained about that.
 
The bin is for binaries. Something that humans don't need to interact with like .dlls, .exes and .runtime.json files.
The users should not be going to the bin to find the results, especially if we are using Crapcade and the binaries are not stored in `.\bin\Release` but in `root\artifacts\bin\$projectName\Release\tfm`. Do you expect the users to know this?

Reply:
I still do not agree, but it is a matter of preference. There is always `DIR /AD /S /B BenchmarkDotNet.Artifacts` and `find . -name BenchmarkDotNet.Artifacts -type d`.
If you still decide to push these change, then you must add the folder name to `.gitignore`.

Reply:
I have reverted  those changes, it's most probably better to have a separate PR for that.

Reply:
FWIW I agree with you @adamsitnik. Having the results buried deeply in the tree never made sense to me.